qa/rpc-tests/remove_sprout_shielding.py

```diff
@@ -41,6 +41,7 @@ def run_test (self):
 
```

Comment:
Corresponding to a comment below, we need to add a test case that we don't generate sprout addresses in the initial block download.

Reply:
It's difficult to do this in a python test, so this test is implemented in the Boost test.